client/chainlink.go

```diff
@@ -62,22 +64,43 @@ func NewChainlink(c *ChainlinkConfig, httpClient *http.Client) (Chainlink, error
 	return cl, cl.SetSessionCookie()
 }
 
+func FundTemplateNodes(blockchainClient BlockchainClient, wallets BlockchainWallets, nodes []Chainlink, ethAmount int64, linkAmount int64) error {
```

Comment:
The client is specifically for API interactions with the Chainlink node, it needs to have no dependencies on anything blockchain related. These sorts of actions that combine clients & blockchain should go into another package.